### DISCUSSION TITLE
Change commands

### DISCUSSION THREAD

**Discussion by robisxx** [jagrosh/MusicBot#1311] [Questions]
Hi.
Is it possible to change commands? 
My prefix is !, so I want to use "!p" instead "!play" and "!s" instead "!skip".

**Comment by hubert4346** [comment]
You can edit this in config file

Find this fragment of config file:


`// These settings allow you to configure custom aliases for all commands.
// Multiple aliases may be given, separated by commas.
//
// Example 1: Giving command "play" the alias "p":
// play = [ p ]
//
// Example 2: Giving command "search" the aliases "yts" and "find":
// search = [ yts, find ]

aliases {
  // General commands
  settings = [ status ]

  // Music commands
  lyrics = []
  nowplaying = [ np, current ]
  play = []
  playlists = [ pls ]
  queue = [ list ]
  remove = [ delete ]
  scsearch = []
  search = [ ytsearch ]
  shuffle = []
  skip = [ voteskip ]

  // Admin commands
  prefix = [ setprefix ]
  setdj = []
  settc = []
  setvc = []

  // DJ Commands
  forceremove = [ forcedelete, modremove, moddelete ]
  forceskip = [ modskip ]
  movetrack = [ move ]
  pause = []
  playnext = []
  repeat = []
  skipto = [ jumpto ]
  stop = []
  volume = [ vol ]
}`

Between square brackets you can type aliases that You want.
Restart application/server or type !shutdown in discord to apply the changes.
